### PR TITLE
#162163166 User account verification via email

### PR DESCRIPTION
**Pull request by alexkayabula**
## What does this PR do?
- The PR adds a feature that enables successfully registered users to verify their accounts via email.
### Description of task to be completed?
- A user must click a link sent to their email after registration to verify their account.
### How should this be manually tested?
- Clone this repository and checkout the branch: 'ft-email-verification-link-162163166'.
- Set up the project.
- Register a new user with a valid email address with Postman.
- Check the email and click the verification link.
### What are the relevant pivotal tracker stories?
[#162163166](https://www.pivotaltracker.com/story/show/162163166)

### CODE REVIEW

**Review by mulondo — APPROVED** [review]
good job

**Review by lindseyme — APPROVED** [review]
Good work

**Review by sekayasin — APPROVED** [review]
Bravo, No conflicts, Good!

**Review by barema4 — APPROVED** [review]
Nice work

File: authors/settings.py [hunk 1]
```diff
@@ -152,5 +152,12 @@
     ),
 }
 
+EMAIL_HOST = os.environ.get("EMAIL_HOST", 'smtp.gmail.com')
+EMAIL_HOST_USER = os.environ.get("EMAIL_HOST_USER", 'seven.zeusgeek@gmail.com')
+EMAIL_HOST_PASSWORD = os.environ.get("EMAIL_HOST_PASSWORD", 'Zeus@admin')
```
Comment (inno-asiimwe) [inline authors/settings.py:157]: Should not put secrets here.

File: authors/apps/authentication/views.py [hunk 2]
```diff
@@ -28,9 +34,27 @@ def post(self, request):
         serializer.is_valid(raise_exception=True)
         serializer.save()
 
-        return Response(serializer.data, status=status.HTTP_201_CREATED)
+        subject = "Hello {}".format(serializer.data['username'] + ", thank you for joining Authors haven")
+        body = "click this link to verify your account   http://localhost:8000/api/users/verified_account/{}".format(
```
[the remark this answers is not in the document]
Comment (alexkayabula) [reply authors/apps/authentication/views.py]: I did implement the changes that you requested

**Pull request merged**